demos/browser/app/meetingV2/meetingV2.ts

```diff
@@ -149,6 +151,8 @@ const VOICE_FOCUS_SPEC = {
   paths: VOICE_FOCUS_PATHS,
 };
 
+const MAX_VOICE_FOCUS_COMPLEXITY: VoiceFocusModelComplexity | undefined = undefined;
```

Comment:
Could you please let us know whether this export needs any documentation or use-case update in README as we now use and showcase this in demo?

Reply:
It's not an export — it's just one of the control flags in the demo, like `LOG_PPS`. I think the code is relatively self-explanatory; there's only a constant because we call with it twice.

Reply:
Sorry for the wrong comment here, I have added a comment where we now import and export the `VoiceFocusModelComplexity` in `index.ts`. Added a new comment [here](https://github.com/aws/amazon-chime-sdk-js/pull/1476/files#r680241342).